estate/models/estate_property.py

```diff
@@ -5,7 +5,7 @@
 class EstateProperty(models.Model):
     _name = "property"
     _description = "Test Model for Training"
-   
+
     name = fields.Char(string = 'Name', required=True)
     description = fields.Text(string = 'Description')
     postcode = fields.Char(string = 'Postcode')
```

Comment:
Remove the string, it is not needed, do the same for all the fields that have a string and is not needed

for example:

"postcode" will be parsed to "Postcode" so the string is not needed at all

let's say we have this field:

"property_state_code_ids" it is a technical name, in that case we want to simplify what it means so:

property_state_code_ids = fields....(string = 'Property region code')

In that case is valid because we want to simplify the string of the field

Reply:
I cannot review on  lines 12 to 43,

so ill leave the comments here:

https://github.com/Cindypsd/Odoo_Training/pull/1/files#diff-26167efdfcbe961332e29e17ced980966bad910db825ec57c717a38b0b591d14R16

this is a good example for when string is needed

comments:

1. in this [line](https://github.com/Cindypsd/Odoo_Training/pull/1/files#diff-26167efdfcbe961332e29e17ced980966bad910db825ec57c717a38b0b591d14R21)

the first parameter of the selection is the technical name the selection will have, change all the first parameter to be more technical (lowercase) e.g. north,south,....